### PR TITLE
Commented code and fixed incorrect photo location bug.

### PR DESCRIPTION
**Pull request by connorasmith**
- Commented a ton of code (finally!)
- Fixed issue with pictures spawning in wrong location in PlacePhotos() method.

### CODE REVIEW

File: Assets/Scripts/LoadPhotos.cs [hunk 1]
```diff
@@ -214,136 +222,192 @@ public void Load()
...
+
+        for(int i = 0; i < photos.Count; i++) {
 
-            LoadPictureLocations(photos.Count);
-            photos[i].transform.position = pictureLocations[i];
+            // Places photo under the photoHolder.
             photos[i].transform.parent = photoHolder.transform;
 
+            // Updates the position of the photo to new location (Should be centered around camera).
+            photos[i].transform.localPosition = pictureLocations[i];
```
Comment (connorasmith) [inline Assets/Scripts/LoadPhotos.cs:410]: Hoping this fixes photo issue.

File: Assets/Scripts/LoadPhotos.cs [hunk 2]
```diff
@@ -3,45 +3,68 @@
 using System.IO;
 using System.Collections.Generic;
 
+/// <summary>
```
Comment (kagcaoili) [inline Assets/Scripts/LoadPhotos.cs:6]: What's a summary?

Comment (connorasmith) [inline Assets/Scripts/LoadPhotos.cs:6]: The summary shows up whenever you try to call this method from somewhere else!

**Comment by kagcaoili** [comment]
accepted!!

**Pull request merged**